### PR TITLE
maint: update the requirements

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**

### CODE REVIEW

**Comment by Revathyvenugopal162** [comment]
> @Revathyvenugopal162 Please check the doc before merging. There has been an issue since #954 with newer pyvista rendering scalar bars wrong Which is why it has been pinned to an older version in `requirements_docs.txt` for a long time. No clue as to the origin of the issue. It is also still the case with PyVista 0.41.1 #1116

Thanks for notifying this @PProfizi , then we can keep pyvista as it is, Since no other docs dependency is direct dependent of pyvista, we can upgrade other requirements especially sphinx and ansys-sphinx theme ( the logo that we have right now in pydpf is older logo)

File: requirements/requirements_docs.txt [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-imageio-ffmpeg==0.4.7
-nbsphinx==0.9.1
+imageio==2.31.1
+imageio-ffmpeg==0.4.8
+nbsphinx==0.9.2
 pypandoc==1.11
 pytest-sphinx==0.5.0
-pyvista==0.36.1
-sphinx==5.3.0
+pyvista==0.41.1
```
Comment (Revathyvenugopal162) [inline requirements/requirements_docs.txt:9]:
```suggestion
pyvista==0.36.1
```

File: requirements/requirements_docs.txt [hunk 2]
```diff
@@ -1,16 +1,16 @@
-ansys_sphinx_theme==0.9.5
-enum-tools[sphinx]==0.9.0.post1
+ansys-sphinx-theme==0.10.2
+enum-tools[sphinx]==0.10.0
 graphviz==0.20.1
-imageio==2.28.0
-imageio-ffmpeg==0.4.7
-nbsphinx==0.9.1
+imageio==2.31.1
```
Comment (Revathyvenugopal162) [inline requirements/requirements_docs.txt:4]:
```suggestion
imageio==2.28.0
```

File: requirements/requirements_docs.txt [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+imageio-ffmpeg==0.4.8
```
Comment (Revathyvenugopal162) [inline requirements/requirements_docs.txt:5]:
```suggestion
imageio-ffmpeg==0.4.7
```

**Comment by PProfizi** [comment]
HI @Revathyvenugopal162 is this still an active topic? I think we can close as stale or not needed anymore.

**Review by jorgepiloto — COMMENTED** [review]
I think we can close this issue, @PProfizi.

**Pull request closed without merging**